charts/policies/user-group-psp/values.yaml

```diff
@@ -0,0 +1,29 @@
+---
+kind: ClusterAdmissionPolicy
+metadata:
+  name: user-group-psp
+  namespace: default # used when kind is AdmissionPolicy
+spec:
+  module:
+    repository: "registry://ghcr.io/kubewarden/policies/user-group-psp"
+    tag: "v0.2.0"
```

Comment:
this `repository` and `tag` fields are the only ones that don't match 1-to-1 with the CRD in the values.yaml.